OpenChange/iCalEvent+MAPIStore.m

```diff
@@ -457,7 +460,14 @@ - (void) updateFromMAPIProperties: (NSDictionary *) properties
                     {
                       [person setParticipationStatus: newPartStat];
                       [person setRsvp: @"TRUE"];
-                      [person setRole: @"REQ-PARTICIPANT"];
+                      if (i < ccFirstIndex)
```

Comment:
You are right @kamenim, I was thinking on doing it that way but finally choose this small change to have a small and less invasive patch, but if no one disagrees I will do that refactor.